<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="locationTitle">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="locationLabel">
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QTextBrowser" name="descBrowser"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="weatherTitle">
     <property name="text">
      <string>Weather:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="weatherLabel">
     <property name="text">
      <string>Weather</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ratingTitle">
     <property name="text">
      <string>Rating:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="ratingLabel">
     <property name="text">
      <string>Rating</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
